sdk/src/lib.rs

```diff
@@ -43,15 +43,16 @@ pub use signer::signers;
 #[cfg(not(target_os = "solana"))]
 pub use solana_program::program_stubs;
 pub use solana_program::{
-    account_info, address_lookup_table_account, alt_bn128, big_mod_exp, blake3, borsh, bpf_loader,
-    bpf_loader_deprecated, bpf_loader_upgradeable, clock, config, custom_heap_default,
-    custom_panic_default, debug_account_data, declare_deprecated_sysvar_id, declare_sysvar_id,
-    decode_error, ed25519_program, epoch_schedule, fee_calculator, impl_sysvar_get, incinerator,
-    instruction, keccak, lamports, loader_instruction, loader_upgradeable_instruction, message,
-    msg, native_token, nonce, program, program_error, program_memory, program_option, program_pack,
-    rent, sanitize, sdk_ids, secp256k1_program, secp256k1_recover, serde_varint, serialize_utils,
-    short_vec, slot_hashes, slot_history, stable_layout, stake, stake_history, syscalls,
-    system_instruction, system_program, sysvar, unchecked_div_by_const, vote, wasm_bindgen,
+    account_info, address_lookup_table_account, address_lookup_table_program, alt_bn128,
```

Comment:
we shouldn't need to modify src/lib.rs. it's not required for solana-sdk to export all symbols from solana-program. The existing symbols are exported for backwards compat only

Reply:
Ah, I see. Wasn't sure if need to add to it, was try to play safe. Will revert.